app/screens/settings-screen/settings-context.tsx

```diff
@@ -0,0 +1,23 @@
+import { PropsWithChildren, createContext, useContext } from "react"
```

Comment:
what is the value of this context? seems we could get along without it

if you're using it to manage the cache, then just reused `useSettingsScreenQuery` with a `fetchPolicy: "cache-only"` for all but the main component?